util/coin/fixtures.go

```diff
@@ -0,0 +1,46 @@
+package coin
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	appparams "github.com/umee-network/umee/v4/app/params"
+	leveragetypes "github.com/umee-network/umee/v4/x/leverage/types"
+)
+
+// common coins used in tests
+var (
+	UumeeDenom = leveragetypes.ToUTokenDenom(appparams.BondDenom)
+	Umee1      = New(appparams.BondDenom, 1)
+	Umee10k    = New(appparams.BondDenom, 10_000)
+	UUmee1     = UUmee(1)
+
+	Umee1dec = DecInt(appparams.BondDenom, 1)
+
+	Atom    = "atom"
```

Comment:
We generally use an ibc denom from fixtures for this, not just "atom" (which would be "uatom" anyway)

Reply:
changing to `uatom`. We can have IBCAtom denom as well.

Reply:
There is a danger, I think, in having our `fixtures` contain two conflicting denoms for atom. (It would be difficult to figure out why a test was failing, for example, if it was due to different atom fixtures being used which were expected to match)

Since the native `uatom` will never exist on our chain, the IBC denom should be used consistently.

Reply:
we are using it:
```
oracle/types/params_test.go
236:	p11.AcceptList[0].BaseDenom = "uatom"

leverage/README.md
360:          "base_denom": "uatom",
```